src/components/CellLineTable/index.tsx

```diff
@@ -74,17 +98,51 @@ const CellLineTable = ({
         };
     };
 
-    const interactiveColumns = columns.map((column: any) => {
+    const applySortingProperties = (column: any) => {
+        const isCurrentlySortedCol = sortedColumn.key === column.key;
+        const sortIcon =
+            isCurrentlySortedCol && sortedColumn.order === "descend" ? (
+                <CaretUpOutlined />
+            ) : (
+                <CaretDownOutlined />
+            );
+        // reverse sort direction for first column so that
+        // it toggles on first selection
+        const sortDirections =
+            column.key === "cellLineId"
+                ? ["descend", "ascend", "descend"]
+                : ["ascend", "descend", "ascend"];
+        return {
+            ...column,
+            title: (
+                <div className={columnHeader}>
+                    {column.title} {sortIcon}
+                </div>
+            ),
+            className: isCurrentlySortedCol ? activeColumn : "",
+            sortDirections: sortDirections,
+        };
+    };
+
+    const interactiveColumns = columns.map((column: any) => {        
         // the two clickable columns are the order cell line and
         // CoA column. They do not have the hover effect and
-        // should not take you to the cell line page
+        // should not take you to the cell line page, and are not
+        // sortable.
         if (column.className?.includes("action-column")) {
             return column;
         }
-        return {
+        const interactiveColumn = {
             ...column,
             onCell: onCellInteraction,
         };
+        if (sortableTable && column.sorter) {
+            return applySortingProperties(interactiveColumn);
+        }
+        return {
+            ...interactiveColumn,
+            sorter: undefined,
```

Comment:
why do you have to do this? shouldn't it already be undefined if it's not a sortable column?